### PR TITLE
Implement DB

### PR DESCRIPTION
**Pull request by yuriyoleynikov**
Addresses #9

### CODE REVIEW

File: TodoListApp/src/TodoListApp/Models/TodoItemTable.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace TodoListApp.Models
+{
+    public class TodoItemTable
```
Comment (Igorbek) [inline TodoListApp/src/TodoListApp/Models/TodoItemTable.cs]: - `.Models` isn't a proper namespace for db data - move to `.Data`
- `TodoItemTable` doesn't sound well tho, rename to `TodoItem`

File: TodoListApp/src/TodoListApp/Startup.cs [hunk 2]
```diff
@@ -39,6 +39,9 @@ public Startup(IHostingEnvironment env)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            var connection = @"Server=(localdb)\mssqllocaldb;Database=TodoListApp;Trusted_Connection=True;";
```
Comment (Igorbek) [inline TodoListApp/src/TodoListApp/Startup.cs]: use a setting instead, see how it's done for `ApplicationDbContext` below

File: TodoListApp/tests/TodoListApp.Tests/InDatabaseRepositoryTests.cs [hunk 3]
```diff
@@ -0,0 +1,540 @@
...
+using Microsoft.EntityFrameworkCore;
+using TodoListApp.Data;
+using Microsoft.Extensions.DependencyInjection;
+using System.Linq;
+
+namespace TodoApp.Services.Tests
+{
+    public class InDatabaseRepositoryTests
+    {
+        private static DbContextOptions<MyDbContext> CreateNewContextOptions()
```
Comment (Igorbek) [inline TodoListApp/tests/TodoListApp.Tests/InDatabaseRepositoryTests.cs]: the next task will be to refactor tests to use the same code. just a heads-up

File: TodoListApp/src/TodoListApp/Data/MyDbContext.cs [hunk 4]
```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using TodoListApp.Models;
+
+namespace TodoListApp.Data
+{
+    public class MyDbContext : DbContext
```
Comment (Igorbek) [inline TodoListApp/src/TodoListApp/Data/MyDbContext.cs]: rename to something more meaningful, like `TodoListDbContext`

File: TodoListApp/src/TodoListApp/Migrations/MyDbContextModelSnapshot.cs [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    [DbContext(typeof(MyDbContext))]
+    partial class MyDbContextModelSnapshot : ModelSnapshot
+    {
+        protected override void BuildModel(ModelBuilder modelBuilder)
+        {
+            modelBuilder
+                .HasAnnotation("ProductVersion", "1.0.1")
+                .HasAnnotation("SqlServer:ValueGenerationStrategy", SqlServerValueGenerationStrategy.IdentityColumn);
+
+            modelBuilder.Entity("TodoListApp.Models.TodoItemTable", b =>
```
Comment (Igorbek) [inline TodoListApp/src/TodoListApp/Migrations/MyDbContextModelSnapshot.cs]: when you rename table, this will also need to be aligned with a new name

**Review by Igorbek — CHANGES_REQUESTED** [review]

File: TodoListApp/src/TodoListApp/Migrations/MyDbContextModelSnapshot.cs (hunk 5 above)
Comment (yuriyoleynikov) [reply TodoListApp/src/TodoListApp/Migrations/MyDbContextModelSnapshot.cs]: done

File: TodoListApp/src/TodoListApp/Data/MyDbContext.cs (hunk 4 above)
Comment (yuriyoleynikov) [reply TodoListApp/src/TodoListApp/Data/MyDbContext.cs]: done

File: TodoListApp/src/TodoListApp/Models/TodoItemTable.cs (hunk 1 above)
Comment (yuriyoleynikov) [reply TodoListApp/src/TodoListApp/Models/TodoItemTable.cs]: done

**Review by Igorbek — APPROVED** [review]

**Pull request merged**